### ISSUE TITLE
Can't Attach Note to Email

### ISSUE THREAD

**Issue by pmruzicka** [xnotepp/xnote#97]
I have the oddest thing happening this morning. I tried to attach a note to an email like I always do and it disappears when I click on another email. I have tried to attach notes to other emails as a test and they work fine. I am clueless as to why the note will not attach to this one specific email.  Is there something I can check to troubleshoot? I have never had this happen before and Thunderbird (102.0.1) and XNote++ (3.3.2) are both up to date.

**Comment by pmruzicka** [comment]
I just saw a related issue where the Message ID has too many characters and that is what I have here:
Message ID = <xserial.s56baew0HaencVmtW022ppoC7zhbX9jLF2+q4Zr7zN6r72e2gW152b1FWhSb4g1JxNLvO0aeri0kbUnmkxmT8BBnAplwdDUaEPmC/21m7wH3vfiQ/Vr6RM3ZCbU2iPrD8+TRnUvMTk4R5OdDmqzhGd/K2Q==@getresponse-mail.com>

So I am assuming this is my issue here.

**Comment by opto** [comment]
yes, at the moment we convert that to real filenames. A database storage will come, where the layout of id can be arbitrary.

with header tools light, you can change the message id, but then new incoming messages may  not be connected to this thread.
